waspc/data/Generator/templates/server/src/utils.js

```diff
@@ -32,7 +32,7 @@ export const prismaErrorToHttpError = (e) => {
   if (e instanceof Prisma.PrismaClientKnownRequestError) {
     if (e.code === 'P2002') {
       return new HttpError(422, 'Save failed', {
-        message: `A record with the same ${e.meta.target.join(', ')} already exists.`,
+        message: `user with the same ${e.meta.target.join(', ')} already exists`,
```

Comment:
Will this always be "user"? Couldn't we get the same error from a failed save in a different context/entity?

Reply:
Makes sense. I'd prob add a TODO so we remember 👍🏻